File: src/test/java/com/technews/scheduler/ReleaseSpringProjectsSchedulerTest.java
package com.technews.scheduler;

import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;

@Disabled
@ActiveProfiles("local")
@SpringBootTest
class ReleaseSpringProjectsSchedulerTest {

    @Autowired
    private ReleaseSpringProjectsScheduler releaseSpringProjectsScheduler;

    @Test
    void run() {
        releaseSpringProjectsScheduler.runScheduler();
    }
}

File: src/test/java/com/technews/scheduler/ReleaseSpringProjectsSchedulerTest.kt
package com.technews.scheduler

import com.technews.IntegrationTest
import io.kotest.core.spec.style.StringSpec
import org.junit.jupiter.api.Disabled
import org.springframework.beans.factory.annotation.Autowired

@Disabled
@IntegrationTest
class ReleaseSpringProjectsSchedulerTest(
    @Autowired private val releaseSpringProjectsScheduler: ReleaseSpringProjectsScheduler,
) : StringSpec({
        "scrape a spring project releases post" {
            releaseSpringProjectsScheduler.runSchedule()
        }
    })
